Docs/Workflows.md

```diff
@@ -62,7 +62,22 @@ jobs:
 
 Checks for spelling mistakes in a repository using the [Check Spelling](https://github.com/marketplace/actions/check-spelling) GitHub Action, proxied by the [`spelling` action](../.github/actions/spelling/action.yml) in this repository, which has [its own documentation](SpellCheckingConfiguration.md) describing the configuration options and contribution guidelines. This documentation is also linked automatically at the end of every spell checking report of a pull request.
 
-Example _check-spelling.yml_:
+### How to integrate spell checking into a project
+
+1. Start by adding a job to an existing workflow or create one specific to spell checking with the minimum configuration.
+2. Push the changes and open a pull request to have the initial spell-checking report commented to it.
+3. Follow the [spell checking configuration tips](SpellCheckingConfiguration.md) to work through the list of unrecognized entries.
+4. You will probably end up with a few configuration files and some external dictionaries applied, so your workflow might end up looking something like this.
+
+#### Minimum spell checking step configuration example
```

Comment:
Grammarly suggests "spell-checking". I didn't find an authoritative answer for the spelling (haha) of this, please check.

Reply:
Grammarly is right here, because you're using this word pair as an [adjective](https://en.wikipedia.org/wiki/Adjective) to "step configuration".

This is the same as saying "We do open source." vs. "We build open-source software."